src/btn.js

```diff
@@ -0,0 +1,17 @@
+function createBtn() {
+  const createBtn = document.createElement('button')
+  createBtn.innerHTML = 'play the game'
+  createBtn.classList.add('first_btn')
+  document.querySelector('#app').appendChild(createBtn)
```

Comment:
this querySelector shouldn't be here. Move it to app.js and set it as variable called `root`.

```js
const root = document.querySelector('#app');

// then use appendChild on root variable to create the button 
```

Reply:
I don't know how to fix it.

Reply:
At first remove the `document.querySelector('#app').appendChild(createBtn)` from this function.
Then you should add the `return` statement, so when we call this function we can assign **whatever it returns** to a variable.